### PR TITLE
[CARBONDATA-383]Optimize hdfsStoreLocation/hdfsStorePath parameters' name since Carbon not only support hdfs path

### PR DESCRIPTION
**Pull request by Hexiaoqiao**
change parameters' name from hdfsStoreLocation/hdfsStorePath to storePath in order to avoid ambiguity.

### CODE REVIEW

**Comment by Hexiaoqiao** [comment]
ci passed: http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/579/

File: integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataLoadRDD.scala [hunk 1]
```diff
@@ -166,7 +166,7 @@ class SparkPartitionLoader(model: CarbonLoadModel,
  * @param result                Output result
  * @param carbonLoadModel       Carbon load model which contain the load info
  * @param storeLocation         Tmp store location
- * @param hdfsStoreLocation     The store location in hdfs
+ * @param storePath             The store location in hdfs
```
Comment (chenliang613) [inline integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataLoadRDD.scala]: the comment need change accordingly of "The store location in hdfs"

**Comment by Hexiaoqiao** [comment]
@chenliang613  thanks for ur comment, i have fixed it. 
Actually there are still some parameters' name include 'hdfs*' such as #CarbonGlobalDictionaryRDD.scala, i think it makes sense using 'hdfs' since it involves with some configurations which specify explicitly hdfs path,  [HDFS_TEMP_LOCATION = "hadoop.tmp.dir"] in CarbonCommonConstants.java for one.

**Comment by chenliang613** [comment]
LGTM

**Comment by ravipesala** [comment]
LGTM

**Comment by ravipesala** [comment]
@Hexiaoqiao Please squash the commits to single commit

**Comment by Hexiaoqiao** [comment]
Thanks @ravipesala for your comments, i squash same commit to single one. 
ci passed: http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/596/

pls help to check if there are some exception.

**Comment by chenliang613** [comment]
@Hexiaoqiao please squash into one commit, there are still some exceptions

**Comment by Hexiaoqiao** [comment]
@chenliang613  DONE, pls help to check. thank you.

**Pull request merged**